tensorflow/python/ops/image_ops_impl.py

```diff
@@ -849,7 +849,9 @@ def central_crop(image, central_fraction):
 
   Remove the outer parts of an image but retain the central region of the image
   along each dimension. If we specify central_fraction = 0.5, this function
-  returns the region marked with "X" in the below diagram.
+  returns the region marked with "X" in the below diagram. Larger the value of 
```

Comment:
Use backticks around symbol names `central_fraction=0.5` and `central_fraction`.

Reply:
Done, Thanks for the review.